pkg/configuration/base/validate.go

```diff
@@ -534,5 +608,9 @@ func (r *JenkinsBaseConfigurationReconciler) validateConfiguration(configuration
 			return messages, stackerr.WithStack(err)
 		}
 	}
+	if resources.GlobalProxy.Status.HTTPProxy != "" {
+		r.CreateGlobalProxyConfigMap(jenkinsInstanceNamespace)
```

Comment:
are we allowed to create a global proxy configuration as  jenkins-operator role?
That's tricky.

Reply:
Considering it's a configmap which is being created, I assume that's the only requirement necessary. But yeah, it does seem a bit tricky